core/graphelier-service/api/hndlrs/orderhandler_test.go

```diff
@@ -0,0 +1,69 @@
+package hndlrs_test
+
+import (
+	"graphelier/core/graphelier-service/models"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"graphelier/core/graphelier-service/api/hndlrs"
+	. "graphelier/core/graphelier-service/utils/test_utils"
+)
+
+func TestFetchOrderInfoSuccess(t *testing.T) {
+
+	mockedDB := MockDb(t)
+	defer Ctrl.Finish()
+	messages := []*models.Message{&models.Message{Direction: -1, Instrument: "test", Type: models.NewOrder, OrderID: 1, Price: 100, ShareQuantity: 10, Timestamp: 1, SodOffset: 1}}
+	mockedDB.EXPECT().
+		GetSingleOrderMessages("test", int64(1579564800000000000), int64(1579651200000000000), int64(1)).
+		Return(messages, nil)
+
+	var orderInfo models.OrderInfo
+	err := MakeRequest(
+		hndlrs.FetchOrderInfo, // Function under test
+		mockedDB,
+		"GET",
+		"/order/test/1/2",
```

Comment:
It would be best to have the URL string match the `mux` params set on the request

Reply:
Woops was a remnant of the copy paste